app/index.js

```diff
@@ -270,94 +278,92 @@ module.exports = yeoman.Base.extend({
       },
       done
     );
-  },
+  }
 
-  copyFiles: function() {
+  writing() {
```

Comment:
nitpick: can there be a better function name here?

Reply:
couldn't remember why I changed the name....reverted it.